db/schema.rb

```diff
@@ -25,12 +25,18 @@
     t.boolean  "is_admin",        default: false
     t.datetime "created_at",                      null: false
     t.datetime "updated_at",                      null: false
-    
     t.string   "remember_digest"
   end
 
   add_index "users", ["email"], name: "index_users_on_email", unique: true, using: :btree
-  
   add_index "users", ["username"], name: "index_users_on_username", unique: true, using: :btree
 
+  create_table "widgets", force: :cascade do |t|
```

Comment:
I have no idea! Git Status was saying I updated schema.rb for some reason even though there were no changes and it wouldn’t let me branch, so i figured I must've changed something by accident. I did git reset and it said I was up to date without any changes so I went from there